airflow/www/templates/airflow/trigger.html

```diff
@@ -0,0 +1,36 @@
+{#
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+#}
+
+{% extends "airflow/master.html" %}
+
+{% block content %}
+    {{ super() }}
+    <h2>Trigger DAG</h2>
+    <form method="POST">
+      <input name="csrf_token" type="hidden" value="{{ csrf_token() }}"/>
+      <input name="dag_id" type="hidden" value="{{ dag_id }}"/>
+      <input name="origin" type="hidden" value="{{ origin }}"/>
+      <div class="form-group">
+        <label for="conf">Configuration JSON (Optional)</label>
+        <input class="form-control" name="conf" type="text"/>
+      </div>
+      <input class="btn btn-primary" type="submit" value="Trigger"/>
+      <button class="btn" onclick="window.history.back(); return false">bail.</button>
```

Comment:
If we are passing `origin` to the template we should use that rather than `history.back` --(that or drop the origin param alltogther.)